fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1666,6 +1666,110 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'securityEvents',
+          function (t) {
+
+            function securityEvents(suite) {
```

Comment:
As I was adding tests, the promise chain got longer and longer, and I hated that I could only fit 2 or so "unit tests" on screen at a time. I wanted to be able to glance at all that was being tested.